packages/changed-elements-react/src/widgets/ChangedElementsWidget.tsx

```diff
@@ -41,8 +43,20 @@ export interface ChangedElementsWidgetProps {
   rootElementRef?: React.Ref<HTMLDivElement>;
   /**Optional. If true will use v2 dialog and will run comparison jobs for faster comparisons @beta.*/
   useV2Widget?: boolean;
-  /**Optional. Supply a link for feedback. Should only be used if v2 is enabled*/
+  /** Optional. Supply a link for feedback. Should only be used if v2 is enabled*/
   feedbackUrl?: string;
+  // Optional. When enabled will toast messages regarding job status. If not defined will default to false and will not show toasts (Only for V2).
```

Comment:
Just for consistency:

```suggestion
  /** Optional. When enabled will toast messages regarding job status. If not defined will default to false and will not show toasts (Only for V2). */
```